docker-compose.yml

```diff
@@ -1,88 +1,108 @@
 # itk-version: 3.2.1
 networks:
-  frontend:
-    external: true
-  app:
-    driver: bridge
-    internal: false
+    frontend:
+        external: true
+    app:
+        driver: bridge
+        internal: false
 
 services:
-  mariadb:
-    image: itkdev/mariadb:latest
-    networks:
-      - app
-    ports:
-      - '3306'
-    healthcheck:
-      test: [ "CMD", "healthcheck.sh", "--connect", "--innodb_initialized" ]
-      start_period: 10s
-      interval: 10s
-      timeout: 5s
-      retries: 3
-    environment:
-      - MYSQL_ROOT_PASSWORD=password
-      - MYSQL_USER=db
-      - MYSQL_PASSWORD=db
-      - MYSQL_DATABASE=db
-      #- ENCRYPT=1 # Uncomment to enable database encryption.
+    rabbit:
```

Comment:
Changes from the standard docker-compose.yml should be added to docker-compose.override.yml to make sure docker-compose.yml is easy overrideable by a new template file.